### ISSUE TITLE
What setting are you using in testing OpenScene?

### ISSUE THREAD

**Issue by argosdh** [runnanchen/Label-Free-Scene-Understanding#1]
Hi, 

I have a question during reading the paper. As you reported in Table 1, OpenScene only got 16.8(mIoU) on semantic segmentation of ScanNet validation set. However, when I went through OpenScene, I found it got 47.5(mIoU). How did you modified OpenScene for so called "fair comparison"? 

Thanks!

**Comment by runnanchen** [comment]
> Hi,
> 
> I have a question during reading the paper. As you reported in Table 1, OpenScene only got 16.8(mIoU) on semantic segmentation of ScanNet validation set. However, when I went through OpenScene, I found it got 47.5(mIoU). How did you modified OpenScene for so called "fair comparison"?
> 
> Thanks!

OpenScene adapts the OpenSeg and LSeg as the 2D backbone, where OpenSeg and LSeg are trained on labelled images and significantly outperform our 2D backbone MaskCLIP. Therefore, for a fair comparison in the label-free setting, we replace OpenScene's 2D backbone with MaskCLIP, which is without training on any labelled images. To this end, OpenScene achieves much lower performance than that reported in the original paper.
